### PR TITLE
add more providers, update yessssms docs for YesssSMS v0.4.1

### PR DESCRIPTION
**Pull request by flowolf**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26874

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/yessssms.markdown [hunk 1]
```diff
@@ -31,6 +47,7 @@ notify:
     username: YOUR_PHONE_NUMBER
     password: YOUR_PASSWORD
     recipient: PHONE_NUMBER_TO_NOTIFY
+    provider: MVNO_PROVIDER
```
Comment (klaasnicolaas) [inline source/_components/yessssms.markdown]: if `provider` is not required then please remove.

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/yessssms.markdown (hunk 1 above)
Comment (flowolf) [reply source/_components/yessssms.markdown]: well, it's an optional parameter, but you need it if you want to use the option. I removed it.

**Comment by frenck** [comment]
You can always provide additional examples in a separate paragraph.

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @flowolf! 👍

**Pull request merged**